src/main/js/helpers/legend.js

```diff
@@ -290,6 +365,26 @@ const loadPieLegendItem = (legendSVG, dataTarget, { hoverHandler }) => {
             hoverHandler(dataTarget, constants.HOVER_EVENT.MOUSE_EXIT)
         );
 };
+/**
+ * Validate and return the legendOptions property
+ *
+ * @private
+ * @param {object} graphConfig - config object of Graph API
+ * @param {object} dataTarget - Data points object
+ * @returns {object} legendOptions - legendOptions for the legend
+ */
+const getDefaultLegendOptions = (graphConfig, dataTarget) => {
+    const legendOptions = getDefaultValue(dataTarget.legendOptions, {
+        showShape: getDefaultValue(graphConfig.showShapes, true),
```

Comment:
So we are connecting the showShapes? Didnt we decide against this to not impact current consumers?

Reply:
I have just set showShape to true to avoid the impact on the existing customers.